src/test/java/org/opensearch/ad/transport/AnomalyResultTests.java

```diff
@@ -8,127 +8,9 @@
  * Modifications Copyright OpenSearch Contributors. See
  * GitHub history for details.
  */
-
+/* @anomaly.detection Commented until we have extension support for hashring : https://github.com/opensearch-project/opensearch-sdk-java/issues/200
```

Comment:
Same as above

Reply:
I have commented this out for the hashring since it set up test nodes and registers handlers for these nodes. The hashring is then used to stub calls to return the test nodes. Since we're temporarily rerouting all requests back to the extension node, most of these test classes dont even apply.

Reply:
Not blocking this PR for the above but we should think about the tests related to HashRing. We can modify the tests for extensionNode.